src/app/Program.cs

```diff
@@ -87,9 +87,6 @@ public static async Task Main(string[] args)
         /// <returns>Only returns when ctl-c is pressed and cancellation token is cancelled</returns>
         static async Task RunKeyRotationCheck(CancellationTokenSource ctCancel)
         {
-            string key = config[Constants.CosmosKey];
-            Console.WriteLine($"Cosmos Key: {key.Substring(0, 5)} ...");
```

Comment:
Need to add "string key = config[Constants.CosmosKey]" line back, or at least I dont see where it's declared any more :)